pkg/services/accesscontrol/roles.go

```diff
@@ -295,7 +295,7 @@ var (
 
 	usagestatsReaderRole = RoleDTO{
 		Name:        "fixed:usagestats:reader",
-		DisplayName: "Usage stats report reader",
+		DisplayName: "Usage report reader",
```

Comment:
Where did you get this name from? 🤔

Reply:
I figured since it was in the `Statistics` folder, that the `stats` was implied.  I can change it back though if desired.